plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/IgniteQueryRunner.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ignite;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.trino.Session;
+import io.trino.plugin.tpch.TpchPlugin;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.QueryRunner;
+import io.trino.tpch.TpchTable;
+import org.intellij.lang.annotations.Language;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.OptionalLong;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.airlift.units.Duration.nanosSince;
+import static io.trino.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.trino.testing.QueryAssertions.assertUpdate;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public final class IgniteQueryRunner
+{
+    private static final Logger log = Logger.get(IgniteQueryRunner.class);
+
+    private static final String IGNITE_SCHEMA = "public";
+
+    public static final String CREATE_CUSTOM = "CREATE TABLE customer (" +
+            "custkey bigint NOT NULL," +
+            "name varchar(25) NOT NULL," +
+            "address varchar(40) NOT NULL," +
+            "nationkey bigint NOT NULL," +
+            "phone varchar(15) NOT NULL," +
+            "acctbal double NOT NULL," +
+            "mktsegment varchar(10) NOT NULL," +
+            "comment varchar(117) NOT NULL )" +
+            "WITH (" +
+            "primary_key = ARRAY['custkey'])";
+
+    public static final String CREATE_NATION = "CREATE TABLE nation (" +
+            "nationkey bigint NOT NULL," +
+            "name varchar(25) NOT NULL," +
+            "regionkey bigint NOT NULL," +
+            "comment varchar(152) NOT NULL )" +
+            "WITH (" +
+            "primary_key = ARRAY['nationkey'])";
+
+    public static final String CREATE_ORDERS = " CREATE TABLE orders (" +
```

Comment:
space is not needed (after "), same below

Reply:
I am not sure i understand correctly,  please help to review again.